pkgs/development/libraries/gensio/default.nix

```diff
@@ -1,21 +1,41 @@
-{ stdenv, lib, fetchFromGitHub, autoreconfHook }:
+{ autoreconfHook
+, fetchFromGitHub
+, fetchpatch
+, lib
+, nix-update-script
+, pkg-config
+, stdenv
+}:
 
 stdenv.mkDerivation rec {
   pname = "gensio";
-  version = "2.1.7";
+  version = "2.2.1";
 
   src = fetchFromGitHub {
     owner = "cminyard";
     repo = pname;
     rev = "v${version}";
-    sha256 = "07m8rbdk05biarc9xskwcx9lghj0dff1msxasfc6hi3jywc3xaih";
+    sha256 = "083khzvyvcgi9j99hbaswglivm9s6dly6spjvisvyacavaybgwgb";
+  };
+
+  patches =  [
+    (fetchpatch {
+      url = "https://github.com/cminyard/gensio/commit/12f6203e6f7aa42172177d7b0870777b605af8d9.patch";
+      sha256 = "19dr4iacccc4il3asdxkag6cj2yc4bxd8p451syfxdm6289rwxic";
+    })
+  ];
+
+  passthru = {
+    updateScript = nix-update-script {
+      attrPath = pname;
```

Comment:
I am not a fan of adding this everywhere. nix-update does this without it.

Reply:
Yes, but `nix-update` is not contained within nixpkgs and the manual references this as a valid update script choice [here](https://nixos.org/manual/nixpkgs/unstable/#ssec-stdenv-attributes).

Reply:
I still don't think that we should add this for every package.